CMSgov/eAPD
# Update logrotate config

Resolves # 4168

### Description
Updating logrotate configuration to prevent logs filling up.

### Significant changes or possible side effects
This is an update to the AMI, could potentially break application/instance

### Steps to manually verify this change

1. Log into instance, check logrotate config
2. Check application is still working

### This pull request is ready to review when


### This pull request can be merged when
